backend/CircleForms/Controllers/Authorization/OAuth/OAuthController.cs

```diff
@@ -58,23 +64,17 @@ public async Task<IActionResult> CompleteAuthentication()
             return Forbid();
         }
 
-        var claims = new List<Claim>
-        {
-            new(ClaimTypes.Name, user.Id.ToString()),
-            new(ClaimTypes.Role, "User")
-        };
-        var id = new ClaimsIdentity(claims, "InternalCookies");
-        var authProperties = new AuthenticationProperties
-        {
-            IsPersistent = true,
-            ExpiresUtc = authResult.Properties?.ExpiresUtc
-        };
-
         var redisDb = _redis.GetDatabase();
         if (!redisDb.SetContains("user_ids", user.Id))
         {
-            if (_usersRepository.Get(user.Id) == null)
+            if (await _usersRepository.Get(user.Id) == null)
             {
+                var superAdminId = _configuration["SuperAdminsId"].Split(",");
```

Comment:
Would you mind using [IOptions](https://docs.microsoft.com/en-gb/aspnet/core/fundamentals/configuration/options) instead as we do with [OsuApiConfig](https://github.com/stanriders/circleforms/blob/6557f132158eaae48e8a3737fa10d7e27597cba4/backend/CircleForms/Startup.cs#L34)?